src/evict/evict_inline.h

```diff
@@ -541,6 +544,23 @@ __wt_evict_needed(WT_SESSION_IMPL *session, bool busy, bool readonly, double *pc
     } else {
         dirty_needed = __wt_evict_dirty_needed(session, &pct_dirty);
         updates_needed = __wti_evict_updates_needed(session, &pct_updates);
+
+        /*
+         * Temporary solution to not do updates and dirty eviction using application threads. Log an
+         * error and panic if the cache is full of updates or dirty pages.
+         */
+        if (ignore_updates_dirty && __wt_conn_is_disagg(session) &&
+          !conn->layered_table_manager.leader) {
+            if (pct_updates > pct_full)
```

Comment:
`pct_full` reflects how much data is in the cache -- i.e., if the cache is half full, `pct_full` will be `50.0`. The comparison here should be to some fixed value -- maybe 95%? or against the clean eviction trigger?